bin/ci/ubuntu/build-and-test.sh

```diff
@@ -37,13 +39,13 @@ if [[ ${BUILD:-scons} == "cmake" ]]; then
   fi
   mkdir -p "build/${CMAKE_TARGET}"
   pushd "build/${CMAKE_TARGET}"
-  cmake ../.. -Dtarget=$CMAKE_TARGET ${CMAKE_EXTRA_ARGS}
-  cmake --build . -- $BUILDARGS
+  time cmake ../.. -Dtarget=$CMAKE_TARGET ${CMAKE_EXTRA_ARGS}
```

Comment:
I didn't think about that. The `time` output is not important enough here to break Jenkins, since it's only intended for manually debugging as opposed to the parsing that Jenkins does. I'm going to try changing the format, but if that doesn't work, I'll remove it.